src/app/components/RelatedContentSection/RelatedContentItem/index.styles.ts

```diff
@@ -6,6 +6,10 @@ export default {
       border: '0.1875rem solid transparent',
       height: '100%',
     }),
+  promoFullWidth: () =>
+    css({
+      width: '100%',
```

Comment:
I wonder if `width` should be around 75% to match cannonical?
https://www.test.bbc.co.uk/naidheachdan/sgeulachdan/c18q7nedn2ko?mode=testData

Reply:
Good shout. I will adjust that to match.